README.md

```diff
@@ -32,3 +32,28 @@ To track specific events, call `track` method.
 
 ### Example
 ***Example here***
+
+### Usage
+
+Listening when user open notification 
+-------------------
+You must call the method setNotificationOpenedHandler where it will work as a callback, 
+there you will receive the info that will bring the push, which is of type Map
```

Comment:
Usually, in these READMEs, the idea (idk if it's a standar as well)  is to avoid writing in first or third person. Then, a new way of write the same as follows:

```
Call the method `setNotificationOpenedHandler`. It works as a callback, then the object that brings the push data is received, which is of type `Map<String, Any>`
```

Reply:
Ty for your comments to fix a couple of misspellings 👨‍🏫